packages/trie/test/trie/checkpoint.spec.ts

```diff
@@ -197,4 +198,84 @@ tape('testing checkpoints', function (tester) {
     t.deepEqual(await trie.get(k1), v1, 'after revert (first CP): v1')
     t.end()
   })
+
+  /*
+    In this educational example, it is shown how operations on a clone of a trie
+    can be copied into the original trie. This also includes pruning.
+    A practical use-case of this could be to take two clones of a trie
+    and apply different operations on it. Based on the outcome, pick one.
+    If this "checkpoint copy trick" (see test for this "trick") is not used,
```

Comment:
Perhaps we could link to the "trick" and to the PRs mentioned below?

Reply:
I'm not sure what you mean here, the test itself is the trick?

Reply:
Or do you mean adding URLs?

Reply:
Yeah I meant adding URLs sorry!